tools/tensorflow_docs/api_generator/reference_resolver.py

```diff
@@ -78,15 +79,11 @@ class ReferenceResolver:
       """,
       flags=re.VERBOSE)
 
-  def __init__(
-      self,
-      *,
-      duplicate_of: dict[str, str],
-      is_fragment: dict[str, bool],
-      py_module_names: Union[list[str], dict[str, str]],
-      link_prefix: Optional[str] = None,
-      physical_path: Optional[dict[str, str]] = None,
```

Comment:
Leave `physical_path` in place; removing it breaks the method as it's still referenced.